website/source/docs/guides/deployment.html.md

```diff
@@ -0,0 +1,184 @@
+
+## 1.2 Network Connectivity
+
+LAN gossip occurs between all agents in a single datacenter with each agent sending a periodic probe to random agents from its member list. The initial probe is sent over UDP every second. If a node fails to acknowledge within `200ms`, the agent pings over TCP. If the TCP probe fails (10 second timeout), it asks configurable number of random nodes to probe the same node (also known as an indirect probe). If there is no response from the peers regarding the status of the node, that agent is marked as down.
+
+The agent's status directly affects the service discovery results. If an agent is down, the services it is monitoring will also be marked as down.
+
+In addition, the agent also periodically performs a full state sync over TCP which gossips each agent’s understanding of the member list around it (node names, IP addresses, and health status). These operations are expensive relative to the standard gossip protocol mentioned above and are synced every 30 seconds. For more details, refer to [Serf Gossip docs](https://www.serf.io/docs/internals/gossip.html)
+
+Datacenter designs may opt for a layer 2 or a layer 3 network. Consul’s gossip protocol uses UDP probes initially to detect the health of its peers. In layer 2 networks, the ARP request will be forwarded to all the devices. If you are running a fairly large cluster (i.e. multi-thousand node Consul cluster), this may create some congestion as the gossip probe request for the nodes is sent approximately once per second. This can be improved by increasing the host ARP table size and ARP cache expiration timer.
+
+Layer 3 restricts the ARP requests to a smaller segment of the network. Traffic between the segments typically traverses through a firewall and/or a router. ACL or firewall rules must be updated to allow the following ports:
+
+|Name|Port|Flag|Description|
+|----|----|----|-----------|
+|Server RPC|8300||Used by servers to handle incoming requests from other agents. TCP only.|
+|Serf LAN|8301||Used to handle gossip in the LAN. Required by all agents. TCP and UDP.|
+|Serf WAN|8302|`-1` to disable (available in Consul 1.0.7)|Used by servers to gossip over the LAN and WAN to other servers. TCP and UDP.|
+|HTTP API|8500|`-1` to disable|Used by clients to talk to the HTTP API. TCP only.|
+|DNS&nbsp;Interface|8600|`-1` to disable||
+
+-> **TIP** As mentioned in the [datacenter design section](#1-1-1-single-datacenter), network areas and network segments can be used to prevent opening up firewall ports between different subnets.
+
+### 1.2.1 RAFT Tuning
+
+Leader elections can be affected by network communication issues between servers. If the cluster spans multiple zones, the network latency between them must be taken into consideration and the `raft_multiplier` must be adjusted accordingly.
```

Comment:
Would be good to link to the config page here:

/docs/agent/options.html#raft_multiplier

Reply:
Fixed in c076ff442